packages/fxa-settings/src/components/InputText/index.tsx

```diff
@@ -48,12 +51,15 @@ export const InputText = ({
   autoFocus,
   maxLength,
   pattern,
+  anchorStart,
+  spellCheck,
+  autoComplete,
 }: InputTextProps) => {
-  const [focussed, setFocussed] = useState<boolean>(false);
+  const [focused, setFocused] = useState<boolean>(false);
```

Comment:
It pained me, and then @pdehaan put this up as an issue https://github.com/mozilla/fxa/issues/14636 and I saw that as a go-ahead to change it. It's not even standard in the UK! It's just weird 😉

Reply:
FWIW, just because I submit odd bugs, don't take that to mean they're correct (or even good) ideas.
But in this case, I'm 💯 and this is just objectively odd spelling. #fightme